src/runner.js

```diff
@@ -132,11 +132,14 @@ exports.run = function(config) {
     // establish ngrok tunnel and resolve ngrok url
     .then(function(proxyHost) {
       if (config.tunnel) {
-        console.log('Connecting tunnel');
-        return Tunnel.connect(proxyHost || config.tunnel.host, config.tunnel.token);
-      } else {
-        return Promise.resolve();
+        console.log('Connecting ngrok tunnel');
+        return Tunnel.connect({ ngrok: { host: proxyHost || config.tunnel.host, token: config.tunnel.token }});
+      }
+      if (config.sauce && config.sauce.launchSauceConnect) {
+        console.log('Connecting sauce tunnel');
```

Comment:
Update log to `Connecting Sauce Connect tunnel`

Reply:
changed